python/manylinux1/scripts/build_grpc.sh

```diff
@@ -16,8 +16,61 @@
 # specific language governing permissions and limitations
 # under the License.
 
-export GRPC_VERSION="1.29.1"
-export CFLAGS="-fPIC -DGPR_MANYLINUX1=1"
+export GRPC_VERSION="1.32.0"
```

Comment:
Please update `python/manylinux201x/scripts/build_grpc.sh` as well.

Reply:
Reverted this actually, since we only need gRPC 1.27 for this functionality to work.